### PR TITLE
Dev - Added docker file

### PR DESCRIPTION
**Pull request by jangap411**
Added `dockerfile` for the backend and frontend and `docker-compose.yml`

### CODE REVIEW

File: backend/middlewares/auth.js
Comment (digitalKuMuL) [inline backend/middlewares/auth.js:1]: add auth middleware not docker file

**Review by digitalKuMuL — APPROVED** [review]
look PR can be merged now @jangap411

**Review by digitalKuMuL — APPROVED** [review]
authentication middleware

File: backend/middlewares/auth.js
Comment (jangap411) [reply backend/middlewares/auth.js:1]: yes, two different commits. auth.js file on a different commit @digitalKuMuL

**Pull request merged**